packages/squiggle-lang/__tests__/Reducer/Reducer_Peggy/Reducer_Peggy_ToExpression_void_test.res

```diff
@@ -0,0 +1,20 @@
+open Jest
+open Reducer_Peggy_TestHelpers
+
+describe("Peggy void", () => {
+  //literal
+  testToExpression("()", "{()}", ~v="()", ())
+  testToExpression(
+    "fn()=1",
+    "{(:$_let_$ :fn (:$$_lambda_$$ [_] {1}))}",
+    ~v="@{fn: lambda(_=>internal code)}",
+    (),
+  )
+  testToExpression("fn()=1; fn()", "{(:$_let_$ :fn (:$$_lambda_$$ [_] {1})); (:fn ())}", ~v="1", ())
+  testToExpression(
+    "fn(a)=(); call fn(1)",
```

Comment:
Would it make sense for call to be a function, and used accordingly, like, ``call(fn(1))``? Having a space between them is unusual for the rest of our syntax.

Reply:
@OAGr I thought having less parenthesis would make it better.  
call(fn(1)) vs call fn(1). And users be aware that it is a keyword. 
Also by making it look like a function that I am deceiving the users because it is not a function but a compiler directive in fact.

Reply:
Some notes about keywords like let, call. Just good to know.
We have an let, but just enters the syntax tree by implication.
let/call like statement prefixes are in fact functions that process expression trees to direct the correct code generation. They are functions that take a statement and return an expression

Reply:
Suggestion: call keyword will disappear in a week with this task list
https://github.com/quantified-uncertainty/squiggle/issues/836#issuecomment-1190543835

I can keep it then as optional for back compatibility. And you can merge this if you want to use void in libraries.

Reply:
That works, thanks!